plugins/GSdx/res/tfx.fx

```diff
@@ -632,37 +625,145 @@ float4 ps_color(PS_INPUT input)
 #endif
 
 #if PS_CHANNEL_FETCH == 1
-	float4 t = fetch_red(int2(input.p.xy));
+	float4 T = fetch_red(int2(input.p.xy));
 #elif PS_CHANNEL_FETCH == 2
-	float4 t = fetch_green(int2(input.p.xy));
+	float4 T = fetch_green(int2(input.p.xy));
 #elif PS_CHANNEL_FETCH == 3
-	float4 t = fetch_blue(int2(input.p.xy));
+	float4 T = fetch_blue(int2(input.p.xy));
 #elif PS_CHANNEL_FETCH == 4
-	float4 t = fetch_alpha(int2(input.p.xy));
+	float4 T = fetch_alpha(int2(input.p.xy));
 #elif PS_CHANNEL_FETCH == 5
-	float4 t = fetch_rgb(int2(input.p.xy));
+	float4 T = fetch_rgb(int2(input.p.xy));
 #elif PS_CHANNEL_FETCH == 6
-	float4 t = fetch_gXbY(int2(input.p.xy));
+	float4 T = fetch_gXbY(int2(input.p.xy));
 #elif PS_DEPTH_FMT > 0
-	float4 t = sample_depth(st_int, input.p.xy);
+	float4 T = sample_depth(st_int, input.p.xy);
 #else
-	float4 t = sample_color(st);
+#define CHUCHU
+	float4 T = sample_color(st);
 #endif
 
-	float4 c = tfx(t, input.c);
+	float4 xC = trunc(input.c * 255.0f + 0.5f); // FIX THIS
```

Comment:
If you're trying to follow the OpenGL/GLSL code, then in addition to changing the colour format to `DXGI_FORMAT_R8G8B8A8_UINT` the corresponding `VS_INPUT` entry must be changed to `uint4`.

Reply:
Nice! That did the trick.